src/apps/frontend/components/flex/flex-item.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from "react";
+
+interface FlexItemProps {
+  children: ReactNode;
+  className?: string;
+}
+
+const FlexItem: React.FC<FlexItemProps> = ({
```

Comment:
I just mentioned, repeating here as well.

`const FlexItem: React.FC<PropsWithChildren<FlexItemProps>> = ({` and remove `children` from `FlexItemProps`

Reply:
Fixed